decentraland/unity-renderer
# fix: all scenes stopped loading when one of them failed

## What does this PR change?

Fixed an exception breaking the flow and an error not following the failure flow properly

## How to test the changes?

Run around the metaverse festival scenes with asset bundles disabled, If one scene fails, others should load correctly.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
